package/libnetwork/src/libnetwork.c

```diff
@@ -28,11 +52,111 @@ typedef struct {
   bool debug;
 } network_progress_t;
 
+static volatile bool shutdown_signaled = false;
+static volatile bool cycle_connection_signaled = false;
+
+static time_t last_pipe_warn_time = 0;
```

Comment:
This would only be a problem if the daemons are running the same process space, right?  I can potentially move these to something more "control loop" specific.

Reply:
Oh yeah, good point. Fine with me then.